### PR TITLE
kommander: Bump yakcl to v0.8.1

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
chore: Bump yakcl to v0.8.1
```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/kommander/requirements.lock.orig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- name: grafana
+  repository: https://mesosphere.github.io/charts/stable
+  version: 4.6.3
+<<<<<<< HEAD
+digest: sha256:909336dd2a4fb292961f2d8316624f342c8a22c232e0e4e23524881eb399f5e3
+generated: "2021-01-27T19:09:11.964727+01:00"
+=======
+digest: sha256:6bb1432a7edf4a64d7b3bff653c163a486fb3cb2491f8d65028ef22c731435ef
+generated: "2021-01-27T10:16:26.120166-08:00"
+>>>>>>> 7adf25959 (chore: Bump yakcl to v0.8.1)
```
Comment (s12chung) [inline stable/kommander/requirements.lock.orig:32]: different version of helm or...?

**Review by s12chung — COMMENTED** [review]
⚠️

File: stable/kommander/requirements.lock.orig (hunk 1 above)
Comment (gracedo) [reply stable/kommander/requirements.lock.orig:32]: lol oops

Comment (gracedo) [reply stable/kommander/requirements.lock.orig]: this was from another kommander chart bump pr merging in after I opened this one, accidentally committed this when resolving conflicts

**Review by TattdCodeMonkey — APPROVED** [review]

**Review by s12chung — APPROVED** [review]

**Pull request merged**